frontend/lib/src/components/input_field.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+
+/*
+  Component for a FormBuilderTextField
+  designed with shadow, border
+*/
+class InputField extends StatelessWidget {
+  const InputField(
+      {super.key, required this.name, this.validator,});
+
+  final String name;
+  final String? Function(String?)? validator;
```

Comment:
put comments here as well on what these parameters are

```suggestion
/** text to display */
  final String name;
 /** .... */
  final String? Function(String?)? validator;
```